rpc/shared/shared/src/main/scala/io/udash/rpc/serialization/ExceptionsRegistry.scala

```diff
@@ -0,0 +1,41 @@
+package io.udash.rpc.serialization
+
+import com.avsystem.commons.serialization.GenCodec
+
+import scala.collection.mutable
+import scala.reflect.ClassTag
+
+trait ExceptionsRegistry {
```

Comment:
Why not? It's a minor change for us, but someone might want to use custom implementation.